### PR TITLE
LPS-203356 Do not allow show categories from other scopes

### PR DESCRIPTION
**Pull request by robertoDiaz**
I'm adding this restriction to avoid show categories from other sites in the web layer.

This is the smallest change needed.

I could use a different approach and create a new finder in the service if you find it more appropiate.

### CODE REVIEW

File: modules/apps/message-boards/message-boards-web/src/main/java/com/liferay/message/boards/web/internal/portlet/action/ActionUtil.java [hunk 1]
```diff
@@ -67,6 +69,13 @@ public static MBCategory getCategory(HttpServletRequest httpServletRequest)
 
 		if (categoryId > 0) {
 			category = MBCategoryServiceUtil.getCategory(categoryId);
+
+			if (category.getGroupId() != themeDisplay.getScopeGroupId()) {
```
Comment (AliciaGarciaGarcia) [inline modules/apps/message-boards/message-boards-web/src/main/java/com/liferay/message/boards/web/internal/portlet/action/ActionUtil.java:73]: Question:

Categories of ancestors (global, parent) sites should be allowed?

Comment (robertoDiaz) [reply modules/apps/message-boards/message-boards-web/src/main/java/com/liferay/message/boards/web/internal/portlet/action/ActionUtil.java:73]: With this approach, no, but it makes a lot of sense.

Comment (AliciaGarciaGarcia) [reply modules/apps/message-boards/message-boards-web/src/main/java/com/liferay/message/boards/web/internal/portlet/action/ActionUtil.java:73]: so, ask change? or "patápalante"

Comment (robertoDiaz) [reply modules/apps/message-boards/message-boards-web/src/main/java/com/liferay/message/boards/web/internal/portlet/action/ActionUtil.java:73]: I'll contact you in slack to analyze this together (in a non public place)

Comment (robertoDiaz) [reply modules/apps/message-boards/message-boards-web/src/main/java/com/liferay/message/boards/web/internal/portlet/action/ActionUtil.java:73]: With this change the portlet will honor it's own scope configuration, so I think this is the right behavior.

**Review by AliciaGarciaGarcia — APPROVED** [review]

**Comment by SylviaLuan** [comment]
ci:test:stable

**Comment by SylviaLuan** [comment]
ci:forward:force

**Pull request closed without merging**